processors/db-processor/src/test/kotlin/net/corda/processors/db/internal/reconcile/db/GroupParametersReconcilerTest.kt

```diff
@@ -115,33 +119,56 @@ class GroupParametersReconcilerTest {
         on { create(eq(avroGroupParameters)) } doReturn groupParameters
     }
 
+    private val reconciler: Reconciler = mock()
+    private val reconcilerWriter: ReconcilerWriter<HoldingIdentity, GroupParameters> = mock()
+    private val reconcilerReader: ReconcilerReader<HoldingIdentity, GroupParameters> = mock()
+    private val reconcilerFactory: ReconcilerFactory = mock {
+        on {
+            create(
+                any(),
+                eq(reconcilerReader),
+                eq(reconcilerWriter),
+                eq(HoldingIdentity::class.java),
+                eq(GroupParameters::class.java),
+                any()
+            )
+        } doReturn reconciler
+    }
+
     private val groupParametersReconciler = GroupParametersReconciler(
         cordaAvroSerializationFactory,
         coordinatorFactory,
         dbConnectionManager,
         virtualNodeInfoReadService,
         jpaEntitiesRegistry,
-        groupParametersFactory
+        groupParametersFactory,
+        reconcilerFactory,
+        reconcilerWriter,
+        reconcilerReader,
     )
 
     @Nested
     inner class BuildReadersPerExistingVNodeTest {
         @Test
-        fun `Successfully build reader`() {
+        fun `Successfully build reader and reconciler`() {
             groupParametersReconciler.updateInterval(1000)
 
             assertThat(groupParametersReconciler.dbReconciler).isNotNull
+            assertThat(groupParametersReconciler.reconciler).isNotNull
         }
 
         @Test
-        fun `Reader is not rebuilt if it already exists`() {
+        fun `Reader and reconciler is not rebuilt if it already exists`() {
             groupParametersReconciler.updateInterval(1000)
-            val original = groupParametersReconciler.dbReconciler
+            val originalDbReconciler = groupParametersReconciler.dbReconciler
+            verify(reconciler, times(1)).start()
```

Comment:
```suggestion
            verify(reconciler).start()
```
Once is the default so no need to specify it explicitly.